doc/user-manual/language/function-definitions.lagda.rst

```diff
@@ -77,6 +77,10 @@ Dot patterns
 A dot pattern (also called *inaccessible pattern*) can be used when
 the only type-correct value of the argument is determined by the
 patterns given for the other arguments.
+A dot pattern is not matched against to determine the result of a
+function call, but it serves as a constraint on the result of matching
```

Comment:
The use of the word "constraint" could suggest that a dot pattern can constrain matching, which it can't.  It is rather that the dot pattern is the _product_ of a successful match against the other patterns, it is a forced value.
Maybe just replace "serves as a constraint on" by "is"?

Reply:
I guess you are using "constrain" in a technical sense, but in an informal sense I think it's certainly true that a dot pattern "constrains" matching, in that an (incorrect) dot pattern can cause matches to fail that would have otherwise succeeded.  Is there a way to express this that is technically correct?

Reply:
Ah, I see what you mean.
Maybe: "but it may serve as documentation of the only possible value at the respective position as determined by the other patterns".